packages/message-box/message-box.scss

```diff
@@ -9,16 +9,13 @@ $message-width: rem(560px);
     max-width: $message-width;
     min-height: $layout-spacing--xl;
     padding: $component-spacing--xl;
+    padding-left: rem(56px);
```

Comment:
Noen god grunn til å ikke bruke en verdi i componentspacing skalaen? Isåfall så tenker jeg den fortjener å bli en variabel med et fornuftig navn :)

Reply:
Det är vad dem är satta till nå och brukas lite överallt, då är det nog bättre och göra en issue där men ordentligt går igenom variablerna.